SQL/2015-08-11-projectConfigTable.sql

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Just an aesthetic thing, but could you remove the blank line at the beginning of this file?